ICAFIX/hcp_fix_multi_run

```diff
@@ -124,6 +128,8 @@ opts_AddConfigOptional '--icadim-mode' 'icadimmode' 'icadimmode' '"default" or "
 
 opts_AddOptional '--processing-mode' 'ProcessingMode' '"HCPStyleData" (default) or "LegacyStyleData"' "controls whether --icadim-mode=fewtimepoints is allowed" 'HCPStyleData'
 
+opts_AddOptional '--fix-only' 'RunFIXOnly' 'TRUE or FALSE' 'If TRUE, it skips running ICA using melodic and proceeds directly with running FIX. A previous ICA solution has to exist!' 'FALSE'
```

Comment:
Consider whether `--skip-ica` or `--reuse-existing-ica` or something might be a better name.  "Only" implies that it can never be combined with other possible process-altering flags (and that if we add another step after fix, `--fix-only` might be expected to skip it, while `--skip-ica` would be expected to run it).

Reply:
I agree that `--skip-ica` is clearer. I changed the code for the new version of the PR.

Reply:
Personally, I like the descriptiveness of `--reuse-existing-ica`, so the user doesn't think they can just skip the ica on the first run.  "reuse" has some parallels to the tICA modes, too.

To be clear, please push the commits that make these changes to the same branch on github, and they will show up here.  Please don't open an additional PR for changes requested on this PR.